getting-started/vs-code-integration/new-project-wizard.md

```diff
@@ -36,25 +38,49 @@ The **Create New Project** wizard provides two options to start you project:
     * **Dashboard**&mdash;A basic dashboard that showcases the Telerik UI TileLayout component and how individual blocks (tiles) can fetch data.
     * **Admin**&mdash;A small app that showcases a dashboard for administrators. The app uses some of the main Telerik UI components like the [Data Grid]({%slug grid-overview%}), [Chart]({%slug components/chart/overview%}), [TileLayout]({%slug tilelayout-overview%}), [Form]({%slug form-overview%}), and more.
 
-## Configuring the Project
+The available templates vary depending on the target framework and hosting model. All of the above templates are available for the Client and Server projects for .NET 6 and .NET 7. The Hybrid projects and the .NET 8 WebApp support only the **Start from Blank App** option.
+
+### Section 2: Select Project Name and Path
+
+This section allows you to type your project name and select the output folder for it.
+
+### Section 3: Select UI for Blazor Version
+
+This section allows you to select the preferred UI for Blazor version. The current latest version is pre-selected by default. Therefore, you must ensure that your system has access to the same version of the UI components - you can achieve this with the [Telerik NuGet feed](../../installation/nuget).
+
+### Section 4: Select Hosting Model
+
+Here you can select the desired hosting model&mdash;[**WebApp**]({%slug getting-started/web-app%}), [**Client** (also: WebAssembly)]({%slug getting-started/client-side%}), [**Server**]({%slug getting-started/server-side%}), or [**Hybrid** (a hybrid Blazor MAUI)]({%slug getting-started/hybrid-blazor%}).
+
+The available target framework options are updated upon selecting the hosting model.
+
+### Section 5: Select Your License Type
 
-The wizard allows you to configure the following options:
+By default, the "Paid" option is selected corresponding to the commercial license type.
```

Comment:
Maybe add another note about the NuGet packages needing to match the license type. Or that trial components show a watermark. Btw, what's the significance of choosing the license type here (we're not having anything like it in Visual Studio)? Does it do anything for people's projects?

```suggestion
By default, **Paid** is selected.
```

Reply:
Both packages deliver the same functionality. I believe this i needed so the correct package reference can be included but this depends on the VSX team.